src/main/java/org/entando/plugin/avatar/config/ConfigServiceConfiguration.java

```diff
@@ -1,16 +1,29 @@
 package org.entando.plugin.avatar.config;
 
 import org.entando.config.ConfigService;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 public class ConfigServiceConfiguration {
 
+    @Value("security.oauth.client.client-id")
```

Comment:
Is there a reason we are not using propertyResolver expressions (e.g. ${asdf.asfd.sfda}) here?

Reply:
I've fixed those, thanks for spotting it